notebooks/labs36_notebooks/test.csv

```diff
@@ -0,0 +1,496 @@
+,id,tweets,labels
```

Comment:
Not sure how many of the labels of the new model is going to be trained on, but lets aim for getting 20% to be testing data. I think it would be best if client see metrics based on a larger sample size.

Reply:
I agree with this!